integration-tests/tests/snapd_1_0_packages_test.go

```diff
@@ -94,17 +94,22 @@ func (s *snapd10PackagesTestSuite) postInteractions() apiInteractions {
 		payload:     s.snapPath,
 		waitPattern: `(?U){.*,"status":"active".*"status":"OK","status_code":200,"type":"sync"}`,
 		waitFunction: func() (string, error) {
-			output, err := genericRequest(s.resource()+"/"+data.BasicConfigSnapName+".sideload", "GET", nil)
+			output, err := genericRequest(&requestOptions{
+				resource: s.resource() + "/" + data.BasicConfigSnapName + ".sideload",
+				verb:     "GET"})
```

Comment:
Conventionally, once you do go over the trouble of breaking into multiple lines I suggest always having the end } or }) on a line by itself. Reads better and produces better diffs once the fields change.